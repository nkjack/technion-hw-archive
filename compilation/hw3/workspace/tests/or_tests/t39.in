void main(int a){
return;
}
