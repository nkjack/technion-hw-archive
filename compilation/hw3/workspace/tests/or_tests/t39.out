---end scope---
main - 0 preconditions
a INT -1
Program has no 'void main()' function
